pkg/terminal/config.go

```diff
@@ -25,6 +25,8 @@ func configureCmd(t *Term, ctx callContext, args string) error {
 		if t.client != nil { // only happens in tests
 			lcfg := t.loadConfig()
 			t.client.SetReturnValuesLoadConfig(&lcfg)
+			t.updateColorScheme()
```

Comment:
It seems to me that these two are always called together and calling only one of them would probably be an error. So they should probably be a single function, which should be called `updateColorizer` or something like that.

Reply:
ACK.  I chose`updateConfig` is that OK?